Mathlib/NumberTheory/ArithmeticFunction.lean

```diff
@@ -77,21 +82,25 @@ section Zero
 
 variable [Zero R]
 
-instance : CoeFun (ArithmeticFunction R) fun _ => ℕ → R :=
-  ZeroHom.hasCoeToFun
+instance : FunLike (ArithmeticFunction R) ℕ fun _ ↦ R :=
+  inferInstanceAs (FunLike (ZeroHom ℕ R) ℕ fun _ ↦ R)
```

Comment:
Please add a porting note about moving from `CoeFun` to `FunLike`.

Reply:
Done!